DDR4 OC Guide.md

```diff
@@ -222,66 +226,72 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
 ### Expected Max Frequency
 * Below are the expected max frequency for some of the common ICs:
 
-  | IC | Expected Max Frequency (MHz) |
-  | :-: | :-------------------------: |
-  | Hynix 8Gb AFR | 3600 |
-  | Hynix 8Gb CJR | 4000<sup>1</sup> |
-  | Hynix 8Gb DJR | 4000+ |
-  | Nanya 8Gb B-die | 4000+ |
-  | Micron 8Gb Rev. E | 4000+ |
-  | Samsung 4Gb E-die | 4000+ |
-  | Samsung 8Gb B-die | 4000+ |
-  | Samsung 8Gb D-die | 4000+ |
-  * <sup>1</sup>CJR is a bit inconsistent in my testing. I've tested 3 RipJaws V 3600 CL19 8GB sticks. One of them was stuck at 3600MHz, another at 3800MHz but the last could do 4000MHz, all at CL16 with 1.45v.
+  | IC | Effective Speed attainable at 1.35V (MT/s) | Expected Max Effective Speed (MT/s) |
```

Comment:
Sources for the frequencies at 1.35V?

Reply:
Micron is based on XMPs. AFR is based off the Hynix datasheet. CJR is based on XMP. B die is based on XMP. E die and D die are based off testing on the German Ryzen Ram OC discord and OC discord. These can be removed it that's desired. This section seems to be causing issues. It can be removed.